config/jobs/common/issue-pr-lifecycle.yaml

```diff
@@ -0,0 +1,120 @@
+periodics:
+- name: ci-gardener-triage-robot-close
+  interval: 1h
+  cluster: gardener-prow-trusted
+  decorate: true
+  annotations:
+    description: Closes rotten issues after 30d of inactivity
+  spec:
+    containers:
+    - image: gcr.io/k8s-prow/commenter:v20220206-831340cbbf
+      command:
+      - /app/robots/commenter/app.binary
+      args:
+      - |-
+        --query=repo:gardener/ci-infra
+        -label:lifecycle/frozen
+        label:lifecycle/rotten
+      - --updated=720h
+      - --token=/etc/github-token/token
+      - --endpoint=http://ghproxy.prow.svc.cluster.local
+      - |-
+        --comment=The Gardener project currently lacks enough active contributors to adequately respond to all issues and PRs.
```

Comment:
I think, you're missing the most important part of the comment: 😄 
`/close`

Same for the others with `/lifecycle [state|rotten]`
Otherwise, the issues/PRs will not be labelled/closed, right?

Reply:
ee709f938ce94fbf5648772fc682ae3a596d65f5 is exactly what I meant 😄